### PR TITLE
Model to json

### PR DESCRIPTION
**Pull request by generationzcode**
I added code to let users directly convert their model to json

### CODE REVIEW

**Comment by gbaned** [comment]
@generationXcode Can you please fix the build failures? Thanks!

**Comment by generationXcode** [comment]
@gbaned I didn't understand what went wrong though..., its just saying exited with error. I hope you're not a bot though... are you???? I've had my suspicions...

File: tensorflow/python/keras/engine/network.py [hunk 1]
```diff
@@ -1035,11 +1035,9 @@ def save(self,
                     signatures, options)
 
   def save_weights(self, filepath, overwrite=True, save_format=None):
-    """Saves all layer weights.
-
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/engine/network.py]: Why did you remove this empty line?

File: tensorflow/python/keras/engine/network.py [hunk 2]
```diff
@@ -1035,11 +1035,9 @@ def save(self,
...
+     """Saves all layer weights.
     Either saves in HDF5 or in TensorFlow format based on the `save_format`
     argument.
-
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/engine/network.py]: Why remove this line?

File: tensorflow/python/keras/engine/network.py [hunk 3]
```diff
@@ -1049,7 +1047,6 @@ def save_weights(self, filepath, overwrite=True, save_format=None):
               (ordered names of weights tensor of the layer).
           - For every weight in the layer, a dataset
               storing the weight value, named after the weight tensor.
-
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/engine/network.py]: Why remove this line?

File: tensorflow/python/keras/engine/network.py [hunk 4]
```diff
@@ -1059,24 +1056,21 @@ def save_weights(self, filepath, overwrite=True, save_format=None):
...
     The TensorFlow format matches objects and variables by starting at a root
     object, `self` for `save_weights`, and greedily matching attribute
     names. For `Model.save` this is the `Model`, and for `Checkpoint.save` this
     is the `Checkpoint` even if the `Checkpoint` has a model attached. This
     means saving a `tf.keras.Model` using `save_weights` and loading into a
     `tf.train.Checkpoint` with a `Model` attached (or vice versa) will not match
     the `Model`'s variables. See the [guide to training
-    checkpoints](https://www.tensorflow.org/guide/checkpoint) for details
+    checkpoints](https://www.tensorflow.org/alpha/guide/checkpoints) for details
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/engine/network.py]: Please don't change the URL to the alpha version, let's keep it to the version at head.

Is the URL at head broken?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Why did you remove empty lines in the docstring? They have significance

**Comment by mihaimaruseac** [comment]
Also, regarding the test errors: click on "Details" to the right and you will see a page with all the information about the failing tests. This PRs breaks all of them, please make sure you run at least pylint (ideally you should run all affected tests) before submitting the PR for review.

I know you didn't run it because in the output you can see:

```
2. do_pylint: Python 3 pylint
  FAIL
```

(and the real error some hundreds of lines above).

PS: @rthadur is a human being just like me and you. [Please be respectful](https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md)

**Comment by generationXcode** [comment]
Ohh, sorry... I didn't mean it that way... @rthadur  I was just wondering

File: tensorflow/python/keras/engine/network.py (hunk 2 above)
Comment (generationXcode) [reply tensorflow/python/keras/engine/network.py]: Seemed redundant at that time

**Comment by gbaned** [comment]
@generationzcode Any update on this PR, please. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Comment by generationzcode** [comment]
Hi, I'm so sorry... I'm kind of busy rn, so is it okay if I take it up in a few weeks?

**Comment by mihaimaruseac** [comment]
If no one else takes it, sure.
